Glyssen/Block.cs

```diff
@@ -259,6 +259,10 @@ public bool MatchesReferenceText
 			}
 		}
 
+		public int Length => GetText(false).Length;
```

Comment:
Might want to add a comment about poor performance and/or an ENHANCE comment. I see that this is a refactoring (re-locating) of existing code, but I it looks like performance could be quite a bit better if we only counted the length of ScriptText elements. Calling GetText is fairly expensive as it has to build the composite string -- all we really need is its length. Note: ChapterAnnouncements would still need special handling if we wanted them to remain accurate.